bin/lib/zopen-build

```diff
@@ -687,15 +679,54 @@ build()
 check()
 {
   checklog="${LOG_PFX}_check.log"
+  results=4 # Unknown results
   if [ -n "${ZOPEN_CHECK_CMD}" ]; then
     printHeader "Running Check"
     runAndLog "${ZOPEN_CHECK_CMD} >\"${checklog}\""
-    if ! runAndLog "${ZOPEN_CHECK_RESULTS_CMD}"; then
-      printError "Check failed. Log: ${checklog}"
+    if command -V "${ZOPEN_CHECK_RESULTS}" >/dev/null 2>&1; then
+      ${ZOPEN_CHECK_RESULTS} "${ZOPEN_ROOT}/${dir}" "${LOG_PFX}"
+      results=$?
     fi
   else
     printHeader "Skipping Check"
   fi
+  case "$results" in
+    0) status="Green" ;; # All tests passed
+    1) status="Blue" ;; # Most tests passed
+    2) status="Yellow" ;; # Most tests failed
+    3) status="Red" ;; # Pretty much non-functional
+    *) status="Unknown" ;; # Skipped or bad return code from check results
+  esac
+  echo "$status" >> "${ZOPEN_INSTALL_DIR}/build.status"
+}
+
+createEnv()
+{
+  cat <<zz >"${ZOPEN_INSTALL_DIR}/.env"
+if ! [ -f ./.env ]; then
+  echo "Need to source from the .env directory" >&2
+  return 0
+fi
+export _BPXK_AUTOCVT=ON
+export _CEE_RUNOPTS="\$_CEE_RUNOPTS FILETAG(AUTOCVT,AUTOTAG) POSIX(ON)"
+export _TAG_REDIR_IN=txt
+export _TAG_REDIR_ERR=txt
+export _TAG_REDIR_OUT=txt
+zz
+
```

Comment:
can we export the right 'HOME' variable, e.g. OPENSSL_HOME ? Or is that too noisy? Would save a lot of append env's for all the libraries.